DogsList/DogsList/Modules/StarWar/Model/PersonManager.swift

```diff
@@ -32,7 +32,7 @@ struct PersonManager {
         
             let task = session.dataTask(with: url) { (data, response, error) in
                 if let error = error {
-                     self.delegate?.didFailWithError(error: error)
+                    // self.delegate?.didFailWithError(error: error)
```

Comment:
do not include in pr commented out code.

Reply:
Ok. Fixed.